core/components/molecules/form/fieldset/fieldset.js

```diff
@@ -6,7 +6,14 @@ import { spacing } from '@auth0/cosmos-tokens'
 import StyledDivider from './divider'
 import Automation from '../../../_helpers/automation-attribute'
 
-const StyledFieldSet = styled.fieldset`
+const FieldSet = props => (
+  <FieldSet.Element {...Automation('form.fieldset')}>
+    <StyledDivider>{props.label}</StyledDivider>
```

Comment:
Should kill this as well

(convert `Divider` into a React component that move the styled part to `Divider.Element`)

Reply:
Whoops I didn't see that. Thanks a lot